src/commands/account/view_account_summary/mod.rs

```diff
@@ -52,10 +52,38 @@ impl ViewAccountSummaryContext {
                     })?
                     .access_key_list_view()?;
 
+                let validators_stake = crate::common::get_validators_stake(&json_rpc_client)?;
+
+                let runtime = tokio::runtime::Builder::new_multi_thread()
+                    .enable_all()
+                    .build()?;
+                let chunk_size = 15;
+                let concurrency = 10;
+
+                let delegated_stake: std::collections::HashMap<near_primitives::types::AccountId, crate::common::NearBalance> = runtime
```

Comment:
Currently, the delegated stake for account_id can be viewed on validators and proposals. I need help to view the delegated stake in the validator whitelist.

Reply:
Ok